### ISSUE TITLE
Where can we download the source code for this article?

### ISSUE THREAD

**Issue by chanmmn** [dotnet/AspNetCore.Docs#22030]
Where can we download the source code for this article? The page does not consist of any picture, it is pretty hard to follow along. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 698a8824-7a38-70d6-c041-0dca4c040217
* Version Independent ID: ae663e33-6a08-0b24-aa19-3fc3c9edccfc
* Content: [ASP.NET Core Blazor templated components](https://docs.microsoft.com/en-us/aspnet/core/blazor/components/templated-components?view=aspnetcore-5.0&source=docs)
* Content Source: [aspnetcore/blazor/components/templated-components.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/components/templated-components.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
We're in the process of updating and upgrading topics and samples.

The existing component for this is here ...

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/common/samples/5.x/BlazorWebAssemblySample/Pages/TemplatedComponents.razor

... and to get that full sample, you should download the ZIP archive of this docs repo and navigate to the sample from there.

The sample experience will be addressed as soon as possible, but it will take a while to get through topic overhauls in progress now.

With regard to pictures, we don't use many because they're time-consuming to update. I'll consider adding images to this topic the next time I work on it.

**Comment by chanmmn** [comment]
Thanks
